_posts/2017-09-11-DevSwap.markdown

```diff
@@ -8,11 +8,11 @@ author: [kana]
 
-Around two years ago, we didn't have product teams. Back then, most engineers worked in a variety of stacks. As our business models changed, so did the way we organized ourselves. Interestingly, the number of engineers has not changed drastically, but the way we work has. We have greater focus now that we are specialized engineers, but that means there aren't as many opportunities to work in other stacks.
+Around two years ago, we didn't have product teams. Back then, most engineers worked in a variety of stacks. As our business models changed, so did the way we organized ourselves. Interestingly, the number of engineers has not changed drastically, but the way we work has. We have greater focus now that we are organized into product teams with specific business goals, but that means there aren't as many opportunities to work in other stacks.
 
-Earlier this summer, the engineer leads received some feedback from our engineers about wanting more cross-team collaboration and knowledge sharing. In our weekly engineering-wide meeting, we get to hear what features each team is working on, but it can sometimes feels like there's missing context. Within our practices, there is a better sense of the details, but usually the work itself is deeply nested in another team.
+Earlier this summer, the engineer leads received some feedback from our engineers about wanting more cross-team collaboration and knowledge sharing. In our weekly engineering-wide meeting, we get to hear what features each team is working on, but it can sometimes feel like there's missing context. Within our practices, there is a better sense of the details, but usually the work itself is deeply nested in another team.
 
-We brainstormed several ideas related to knowledge sharing and cross-team collaboration. What stood out the most was that being in product teams meant less opportunities to work in other parts of the business. This led to the question: What would happen if we redistributed ourselves into entirely new teams? Today I'll introduce something new we tried, something we're calling DevSwap.
+We brainstormed several ideas related to knowledge sharing and cross-team collaboration. What stood out the most was that being in product teams meant fewer opportunities to work in other parts of the business. This led to the question: What would happen if we redistributed ourselves into entirely new teams? Today I'll introduce something new we tried, something we're calling DevSwap.
```

Comment:
Maybe a more impactful punchline could be something affirmative, like "Last week, every engineer went to work for a different team for two full days. We called it DevSwap." And in its own paragraph. 

Another way to do this is to start the post with that, kind of like the opening statement that awakens everyone.